schemas/string_interpolation/transforms/telephone_number_link.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/telephone_number_link.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "telephone_number_link": {
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["telephone_number_link"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "telephone_number": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "pattern": "^[0-9 ]+$"
```

Comment:
I don't think the ref can be combined with the regex. This still allows letters, changing `$ref` to `type: string` fixes the issue (and doesn't allow empty strings).